### PR TITLE
Integration/pno

### PR DESCRIPTION
**Pull request by jmwhorton**
Additions and updates for the integration of PNO terms into IAO:
(issue: #237 )

Added IAO:0020000 identifier.
Added IAO:0020001 grapheme.
Added IAO:0020010 identifier creating process.
Added IAO:0020015 personal name.
Added IAO:0020016 given name.
Added IAO:0020017 family name.
Added IAO:0020020 code set.
Moved IAO:0000579 (centrally registered identifier registry) under 'code set'.

### CODE REVIEW

File: iao.owl
Comment (cmungall) [inline iao.owl:6975]: This doesn't resolve.

File: iao.owl
Comment (cmungall) [inline iao.owl:7012]: I recommend all logical definitions have a genus even if this is redundant with domain/range

File: iao.owl
Comment (cmungall) [inline iao.owl:7044]: note the genus in the text and the label of the genus don't match

File: iao.owl
Comment (cmungall) [inline iao.owl:7391]: consider adding this in the imports

**Review by cmungall — CHANGES_REQUESTED** [review]
Necessary changes:

- You should edit https://github.com/information-artifact-ontology/IAO/blob/master/src/ontology/iao-edit.owl not the release file
- Fix the BFO ID that doesn't resolve

All other comments are advisory only

File: iao.owl
Comment (zhengj2007) [reply iao.owl:7391]: Hi @jmwhorton, if you are not sure how to import OBO terms to IAO in the import-OBO.owl file, I can help with it.

Comment (jmwhorton) [reply iao.owl:7391]: Thank you, @zhengj2007.  I believe I have figured it out.

File: iao.owl
Comment (jmwhorton) [reply iao.owl:6975]: I'm not sure what to use here.  I can't find a working resource for BFO 'entity.'  The link below also has the same unresolvable url.  Can you point me in the right direction?

https://www.ebi.ac.uk/ols/ontologies/bfo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000001&lang=en&viewMode=All&siblings=false

File: iao.owl
Comment (zhengj2007) [reply iao.owl:7391]: Both 'organism' and 'Homo sapiens' need go to the import-OBO.owl. The IAO using ontoFox to extract the terms from external ontologies. Please add the terms in OntoFox-input.txt and run ontoFox tool to generate the import-OBO.owl.

File: iao.owl
Comment (zhengj2007) [reply iao.owl:6975]: @cmungall I am not quite sure what you mean. 
I'd prefer to change the axioms as: equivalent:  'information content entity' and (is_specified_output_of some 'identifier creating process') and subClassOf: denotes some entity

File: iao.owl
Comment (jmwhorton) [reply iao.owl]: Genus added to definition (moved to iao-edit.owl)

File: iao.owl
Comment (jmwhorton) [reply iao.owl]: Added import to OntoFox-input.txt

**Comment by zhengj2007** [comment]
All edits look good to me except 'family name' and 'given name'. Both of them are subClass of 'personal name' and also part_of 'personal name'. Does not seem correct.

**Comment by jmwhorton** [comment]
Per conversation with @zhengj2007 and @mbrochhausen, 'family name' and 'given name' have been updated.  Please let me know if the new definition for 'family name' is acceptable (updated to reflect change in hierarchy).

**Comment by jmwhorton** [comment]
Changes have been made per the 2022-07-18 OBI call.

@alanruttenberg, I do have a question for you.  Based on your provided definition of 'code set,' I left it under 'information content entity' because it was worded as such.  Please, let me know if the final intention was to move 'code set' under 'data set,' and I can get that change made.

**Comment by alanruttenberg** [comment]
My recollection was that the discussion had it that it would not be under data set.

**Review by zhengj2007 — CHANGES_REQUESTED** [review]
Better to associated tracker, https://github.com/information-artifact-ontology/IAO/issues/237, using IAO_0000233 'term tracker item', with all the terms added/changed in the pull request.

Update definitions of 'personal name', 'family name' and 'given name' using Aristotle format. For example, personal name: 'An identifier referring to an individual entity that is ascribed personhood by the user of the identifier'.

Did not find term 'IAO:0020002 utterance'

Question: why move the 'family name' as sibling of 'person name' but still keep 'given name' as child of 'person name'?

Add Alan Ruttenberg as term editor of 'code set'. You can also add your name as term editor for the terms you made changes.

**Comment by jmwhorton** [comment]
@zhengj2007, the requested changes have been made.  Thank you for your time.

1. IAO_0000233  has been added to all new and updated terms.
2. Personal, family, and given name definitions have been updated to correct format.
3. Utterance was removed per the  2022-07-18 OBI call.  #237 has been updated to reflect decisions.
4. Family name was moved (per the 2022-07-18 OBI call) because it refers to the name of a family, and does not refer to a person (like a 'personal name').  A 'given name' refers to an individual person, making it still a subclass of 'personal name'.   (This is to the best of my recollection. I would welcome a better explanation if I've misrepresented the reason here.)
5. 'Term editor' annotations have been added.

**Review by zhengj2007 — CHANGES_REQUESTED** [review]
Remove 'identifier' from ontoFox input file and rerun ontoFox to update import_OBO.owl since the term has added in iao_edit.owl
Also update the definition of 'grapheme' and 'identifier creating process' to start with its parent label. 
Thanks!

**Comment by jmwhorton** [comment]
@zhengj2007, the requested changes have been made.  Thanks!

**Review by zhengj2007 — APPROVED** [review]

**Comment by jmwhorton** [comment]
Good afternoon, @cmungall.  When you have a moment, we just need one more review on these changes before merging them.  Thanks!

**Comment by mbrochhausen** [comment]
I think we agreed to have it be a sibling to data set.

**Comment by StroemPhi** [comment]
Hi, when can we expect this change to be released?

**Comment by zhengj2007** [comment]
I am going to make a new release as soon as possible.

**Comment by StroemPhi** [comment]
Thank you @zhengj2007  🚀

**Pull request merged**
